### DISCUSSION TITLE
RetrieveURL

### DISCUSSION THREAD

**Discussion by mradosovic** [fo-dicom/fo-dicom#1174] [Q&A]
I have a problem reading DicomTag.RetrieveUrl on a client.
I'm using Weasis to search local DICOM server and it works fine, but to be able to do WADO import after search is done, Weasis probably needs RetrieveURL to exist in the DataSet. 
Although the value is added to a DataSet, when I check it with WireShark, it's value is unknown.
I have tried to force VR type like this:
resultDataSet.AddOrUpdate(DicomVR.UT, DicomTag.RetrieveURL, $"http://cite.com/api/studies/{study.StudyInstanceUID}");

Can anyone help with a suggestion or similar experience?

**Comment by gofal** [comment]
see issue #1177. this issue is solved already
